libraries/AP_Scripting/docs/docs.lua

```diff
@@ -2494,6 +2504,11 @@ function rangefinder:max_distance_cm_orient(orientation) end
 ---@return integer
 function rangefinder:distance_cm_orient(orientation) end
 
+-- desc
```

Comment:
Good point. Just because none of the other functions in the area have descriptions doesn't mean mine shouldn't. At first, I thought this file was auto generated. The parser will generate this file. But after doing that I realized there were a lot of manual changes in this file.